Flepper.Tests.Unit/QueryBuilder/Commands/WhereFilterTests.cs

```diff
@@ -238,5 +238,168 @@ public void ShoulContainWhereWithNotEqualToNull()
                 .Should()
                 .Contain("WHERE [field] IS NOT NULL");
         }
+
+        [Fact]
+        public void ShoulContainLikeContains()
+        {
+            QueryResult result = FlepperQueryBuilder.Select()
+                .From("table")
+                .Where("field").Contains("abc")
+                .BuildWithParameters();
+
+            result.Query
+                .Trim()
+                .Should()
+                .Contain("LIKE");
```

Comment:
Remove those lines, 250...253, the 258 is enough!
Repeat this same logic to other tests.

Reply:
Ok